pkg/security/secl/model/model.go

```diff
@@ -864,6 +870,7 @@ func (p *ProcessContext) HasParent() bool {
 // ProcessContext holds the process context of an event
 type ProcessContext struct {
 	Process
+	Source uint64 `field:"-" json:"-"`
```

Comment:
I found the source information valuable when looking at generated events. Could you add this field to the `ProcessContextSerializer` as well?

Reply:
sure !